firefox_desktop/firefox_desktop.model.lkml

```diff
@@ -82,3 +82,146 @@ explore: +metrics {
     ]
   }
 }
+
+view: +clients_daily_table__contextual_services_quicksuggest_block_nonsponsored_bestmatch_sum {
+  dimension: value {
+    hidden: yes
+  }
+
+  measure: count {
+    sql: ${TABLE}.value ;;
```

Comment:
You'll need to do some aggregation here
```suggestion
   sql: SUM(${TABLE}.value) ;;
```
For the awesome_bar view the underlying data was already aggregated/grouped but that doesn't seem to be the case here.

Reply:
This is also necessary for all the other views